Analysis/HiggsTauTau/src/HTTCategories.cc

```diff
@@ -834,6 +1263,7 @@ namespace ic {
     wt_em_qcd_down_ = 1.0;
     wt_em_qcd_up_ = 1.0;
     wt_nlo_pt_ = 1.0;
+    wt_tau_id_sf_ = 1.0;
```

Comment:
I was adding it to the even in HTTWeights previously but it's now no longer added so i'll remove this from HTTCategories